src/python/pants/backend/codegen/thrift/java/java_thrift_library.py

```diff
@@ -84,7 +84,6 @@ def default_java_namespace(self):
   def include_paths(self):
     return self._include_paths
 
-  # TODO(Eric Ayers) As of 2/5/2015 this call is DEPRECATED and should be removed soon
   @property
```

Comment:
what is marker mixins? What do you mean by "look for the class"? I tried to use isintance in JvmTarget, but since it requires import of JavaThriftTarget, it leads circular import I think.